src/App/App.js

```diff
@@ -43,6 +44,12 @@ function App() {
             <Auth/>
           </Route>
 
+          {user &&
+            <Route path="/settings">
+              <Settings/>
+            </Route>
+          }
+
```

Comment:
It isn't way. Can you please create component like PrivateRoute and use it here?